x-pack/plugins/spaces/server/lib/get_spaces_usage_collector.ts

```diff
@@ -17,17 +18,20 @@ import { KIBANA_SPACES_STATS_TYPE } from '../../common/constants';
  * @param {boolean} spacesAvailable
  * @return {UsageStats}
  */
-async function getSpacesUsage(callCluster: any, server: Server, spacesAvailable: boolean) {
+async function getSpacesUsage(
+  callCluster: any,
```

Comment:
can be more specific with New platform typings, we know that we ScopedClusterClient.* is passed here